src/PKSim.Core/Model/Simulation.cs

```diff
@@ -646,14 +646,24 @@ public virtual ModelConfiguration ModelConfiguration
       }
 
       /// <summary>
-      ///    Returns the events used in the simulation configuration
+      ///    Returns the event mappings used in the simulation configuration
       /// </summary>
       public virtual EventProperties EventProperties
       {
          get => Properties.EventProperties;
          set => Properties.EventProperties = value;
       }
 
+
+      /// <summary>
+      ///    Returns the observer set mappings used in the simulation configuration
+      /// </summary>
+      public virtual ObserverSetProperties ObserverSetProperties
```

Comment:
The mapping will be saved in the simulation configuraiton (multiple ObserverSet per simulation)